bl/evaluation/non_interactive_evaluator.go

```diff
@@ -0,0 +1,43 @@
+package evaluation
+
+import (
+	"github.com/datreeio/datree/bl/validation"
+)
+
```

Comment:
okay :)

Reply:
I didn't change the property names in lower case because `yaml.Marshal`, `json.Marshal` and `xml.Marshal` do not support field names that start with lower cases. See [this](https://pkg.go.dev/gopkg.in/yaml.v2#Marshal).

Instead I used struct tag to do this.